### PR TITLE
HRSystem updates, Report makers updated, interview makers updated

### PR DESCRIPTION
**Pull request by hermonen**

### CODE REVIEW

File: main/Entities/Intern.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
     */
     private final String internName;
     private final int internAge;
-    private final HashMap<String, Integer> internSkills;
+    private final HashMap<String, Double> internSkills;
```
Comment (Maryyeah) [inline main/Entities/Intern.java:14]: The internSkills should be a hash map with String, Integer. I think merge from main to solve this problem..?

File: test/MonthReportMakerTest.java [hunk 2]
```diff
@@ -1,162 +0,0 @@
-import ControllersPresenters.ReportLevel;
```
Comment (Maryyeah) [inline test/MonthReportMakerTest.java:1]: Merge main so that you don't delete the Tests here.

**Review by Maryyeah — CHANGES_REQUESTED** [review]
Just two things: fix the deleting Tests Problem, and also the InternSkills - double or integer?

File: main/Entities/Intern.java (hunk 1 above)
Comment (hermonen) [reply main/Entities/Intern.java:14]: It was initially Integer, but there were some tests that used a double, and also the textfiles for internSkills contained double values, so just to be safe I changed that to double

File: test/MonthReportMakerTest.java (hunk 2 above)
Comment (hermonen) [reply test/MonthReportMakerTest.java:1]: thank you, will do!

File: main/Entities/Intern.java (hunk 1 above)
Comment (Maryyeah) [reply main/Entities/Intern.java:14]: okok! double it is

**Review by Maryyeah — APPROVED** [review]
The deleted tests are gone. I can merge now yea?

**Pull request merged**